### PR TITLE
Chore/node 8.10.1 pre upgrade

### PR DESCRIPTION
**Pull request by mirceahasegan**
# Context
LW-10280
LW-10548
Upgrade to:
- Cardano Node [8.10.1-pre](https://github.com/IntersectMBO/cardano-node/releases/tag/8.10.1-pre)
- DbSync [sancho-4-2-1](https://github.com/IntersectMBO/cardano-db-sync/releases/tag/sancho-4-2-1)
- Ogmios [6.3.0](https://github.com/CardanoSolutions/ogmios/releases/tag/v6.3.0)

# Proposed Solution
- [x] upgrade node, db-sync
- [x] update configuration files
- [x] update local-network scripts
- [x] regenerate fixtures
- [x] validate all e2e pass
- [x] sanchonet successfully starts

# Important Changes Introduced

### CODE REVIEW

**Comment by mirceahasegan** [comment]
@michalrus , can you generate the corresponding versions for arm64? Thank you.

File: compose/aarch64.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 services:
   cardano-db-sync:
-    image: ghcr.io/input-output-hk/ogmios-tracker/cardano-db-sync:${CARDANO_DB_SYNC_VERSION:-sancho-4-0-0}
+    image: ghcr.io/input-output-hk/ogmios-tracker/cardano-db-sync:${CARDANO_DB_SYNC_VERSION:-sancho-4-2-1}
```
Comment (michalrus) [inline compose/aarch64.yml:5]: Published: [this image](https://github.com/input-output-hk/ogmios-tracker/pkgs/container/ogmios-tracker%2Fcardano-db-sync/205957380?tag=sancho-4-2-1).

File: compose/aarch64.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
   cardano-node:
     image: ghcr.io/input-output-hk/ogmios-tracker/cardano-node:${CARDANO_NODE_VERSION:-8.9.1}
```
[the remark this answers is not in the document]
Comment (mirceahasegan) [reply compose/aarch64.yml]: @michalrus , can you create a 8.10.1-pre image pls: [b9a4205](https://github.com/input-output-hk/cardano-js-sdk/pull/1222/commits/b9a4205c988894d158ba3b7640df6e0c62c2c5bb). 🙇‍♂️

File: packages/e2e/local-network/Dockerfile [hunk 3]
```diff
@@ -5,8 +5,8 @@ FROM ubuntu:${UBUNTU_VERSION} as builder
 ENV DEBIAN_FRONTEND=nonintercative
 
 WORKDIR /build
-ARG CARDANO_NODE_BUILD_URL=https://github.com/IntersectMBO/cardano-node/releases/download/8.9.1/cardano-node-8.9.1-linux.tar.gz
-ARG CARDANO_NODE_BUILD_URL_ARM64=https://github.com/input-output-hk/ogmios-tracker/releases/download/0.1.0/cardano-node-8.9.1-aarch64-linux.tar.gz
+ARG CARDANO_NODE_BUILD_URL=https://github.com/IntersectMBO/cardano-node/releases/download/8.10.0-pre/cardano-node-8.10.0-linux.tar.gz
+ARG CARDANO_NODE_BUILD_URL_ARM64=https://github.com/input-output-hk/ogmios-tracker/releases/download/0.1.0/cardano-node-8.10.0-aarch64-linux.tar.gz
```
[the remark this answers is not in the document]
Comment (mirceahasegan) [reply packages/e2e/local-network/Dockerfile]: superseeded by [b9a4205](https://github.com/input-output-hk/cardano-js-sdk/pull/1222/commits/b9a4205c988894d158ba3b7640df6e0c62c2c5bb)

File: compose/aarch64.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (michalrus) [reply compose/aarch64.yml]: Sure:
* [cardano-node](https://github.com/input-output-hk/ogmios-tracker/pkgs/container/ogmios-tracker%2Fcardano-node/216484125?tag=8.10.1-pre)
* [cardano-submit-api](https://github.com/input-output-hk/ogmios-tracker/pkgs/container/ogmios-tracker%2Fcardano-submit-api/216484515?tag=8.10.1-pre)

**Review by AngelCastilloB — APPROVED** [review]
LGTM @mirceahasegan 🏰

**Review by iccicci — APPROVED** [review]
Excellent! Thank you @mirceahasegan ! 🚀

**Pull request merged**